app/src/main/java/org/oppia/app/player/state/itemviewmodel/ContentViewModel.kt

```diff
@@ -1,4 +1,12 @@
 package org.oppia.app.player.state.itemviewmodel
 
+import androidx.databinding.ObservableField
+
 /** [StateItemViewModel] for content-card state. */
-class ContentViewModel(val htmlContent: CharSequence, val gcsEntityId: String) : StateItemViewModel(ViewType.CONTENT)
+class ContentViewModel(
+  val htmlContent: CharSequence,
+  val gcsEntityId: String
```

Comment:
Can we pass `hasBlueBackground ` and `isCenterAligned ` directly inside the constructor similar to `htmlContent` and `gcsEntityId`.

Reply:
updated as per your suggestion. PTAL